tests/phpunit/tests/interactivity-api/wpInteractivityAPIDirectivesProcessor.php

```diff
@@ -778,4 +778,51 @@ public function test_next_balanced_tag_closer_tag_on_closing_tag() {
 		$p->next_tag( array( 'tag_closers' => 'visit' ) );
 		$this->assertFalse( $p->next_balanced_tag_closer_tag() );
 	}
+
+	/**
+	 * Tests that skip_foreign_content skips to the next tag,
+	 * independant of the content.
+	 *
+	 * @ticket 60517
+	 *
+	 * @covers ::skip_foreign_content
+	 */
+	public function test_skip_foreign_content() {
+		$content = '<div><span>Not closed</div>';
+		$p       = new WP_Interactivity_API_Directives_Processor( $content );
+		$p->next_tag();
+		$this->assertTrue( $p->skip_foreign_content() );
```

Comment:
There's actually no foreign content here. Should a better name be chosen than `skip_foreign_content()` like `jump_to_tag_closer()`?